docs/application/dotnet/tutorials/app-filtering.md

```diff
@@ -168,6 +168,7 @@ The following tables show the available requirements for an application package.
 | `http://tizen.org/feature/opengles.version.3_0` | Specify this key, if the application requires OpenGL&reg; ES version 3.0. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 2.4   |
 | `http://tizen.org/feature/opengles.version.3_1` | Specify this key, if the application requires OpenGL&reg; ES version 3.1. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 3.0   |
 | `http://tizen.org/feature/opengles.version.3_2` | Specify this key, if the application requires OpenGL&reg; ES version 3.2. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 4.0   |
+| `http://tizen.org/feature/opengles.surfaceless_context ` | Specify this key, if the application requires the EGL extension of Surfaceless Context for OpenGL® ES. | 6.0   |
```

Comment:
In the table, please follow the alphabetical order. I suggest moving this feature to the front of  `http://tizen.org/feature/opengles.version.1_1`

```suggestion
| `http://tizen.org/feature/opengles.surfaceless_context` | Specify this key, if the application requires the EGL extension of Surfaceless Context for OpenGL&reg; ES. | 6.0   |
```

@annie-samsung Do you have any comment for this update?

Reply:
@mijin85cho, no further comments from my side.

Reply:
@annie-samsung Thanks for checking.